riptide-opentelemetry/README.md

```diff
@@ -0,0 +1,117 @@
+# Riptide: OpenTelemetry
+
+[![Spider web](../docs/spider-web.jpg)](https://pixabay.com/photos/cobweb-drip-water-mirroring-blue-3725540/)
+
+[![Javadoc](https://www.javadoc.io/badge/org.zalando/riptide-micrometer.svg)](http://www.javadoc.io/doc/org.zalando/riptide-micrometer)
+[![Maven Central](https://img.shields.io/maven-central/v/org.zalando/riptide-opentelemetry.svg)](https://maven-badges.herokuapp.com/maven-central/org.zalando/riptide-opentelemetry)
+
+*Riptide: OpenTelemetry* adds sophisticated [OpenTelemetry](https://opentelemetry.io/) support to *Riptide*.
+
+## Example
+
+```java
+Http.builder()
+    .plugin(new OpenTelemetryPlugin(tracer))
+    .build();
+```
+
+## Features
+
+- Span context injection into HTTP headers of requests
+- Extensible span decorators for attributes
+- Seamless integration with [Riptide: Failsafe](../riptide-failsafe)
+
+## Dependencies
+
+- Java 8
+- Riptide Core
+- [OpenTelemetry Java API](https://opentelemetry.io/docs/instrumentation/java/)
```

Comment:
Being completely accurate, shouldn't we also mention `opentelemetry-semconv`?

Reply:
I think it would be overkill. Think about this like "we depending on spring boot", but not specifying that we depend on spring-context, spring-web-starter etc

Reply:
Fair enough.